share/completions/lsblk.fish

```diff
@@ -1,5 +1,5 @@
 function __fish_print_lsblk_columns --description 'Print available lsblk columns'
-    lsblk --help | sed '1,/Available columns:/d; /^$/,$d; s/^\s\+//; s/\s/\t/'
+    lsblk --help | sed '1,/Available output columns:/d; /^$/,$d; s/^\s\+//; s/\s/\t/'
```

Comment:
This should be optional, for people using older versions.

Also, it's localized, so this won't actually work for people not using an english locale - this needs to be `LC_ALL=C lsblk --help`.

I'd probably make it

```fish
LC_ALL=C lsblk --help | sed '1,/Available .*columns:/d; /^$/,$d; s/^\s\+//; s/\s/\t/'
```

Reply:
Thanks; I will apply this.